fuzzing/fuzz.cpp

```diff
@@ -44,6 +44,10 @@ int main(int argc,char **argv)
 #endif
 	
 	auto packed{readFile(argv[1])};
+	if (!packed)
```

Comment:
Yeah, it was stupid missing check

Reply:
It was fine with the static buffer of size 0 being returned, but `StaticBuffer<0>` is technically not valid C++ :)

Reply:
Anyway, thanks for merging so quickly!